src/parsers/rdp.rs

```diff
@@ -237,13 +237,21 @@ macro_rules! impl_rdp {
                 self.fail_pos = 0;
             }
 
+            fn queue_debug(&self) -> Vec<(Token<Rule>, String)> {
+                self.queue
+                    .clone()
+                    .into_iter()
+                    .map(|t| (t, self.input().slice(t.start, t.end).to_string()))
```

Comment:
that's interesting, I find `to_string` more explicit myself. I can change it though